src/gatling/scala/config/RootConfig.scala

```diff
@@ -19,4 +19,9 @@ class RootConfig(@JsonProperty("tenantId") _tenantId: String,
   val accountManagementApi = _accountManagementApi
   val channelManagementApi = _channelManagementApi
   val publicationServicesApi = _publicationServicesApi
+  val rampUpUsers: Int = Integer.getInteger("rampUsers", 0).toInt
```

Comment:
Not sure these are needed. Could you try to remove?

Reply:
We are using this in simulations